third_party/openmetadata/install_OM.sh

```diff
@@ -1,39 +1,82 @@
 #!/bin/bash
 
+OPERATION=install
 OM_VERSION=0.12.1
+K8S_TYPE=kind
 export OPENMETADATA_AIRFLOW_VERSION=0.12.1
 export OPENMETADATA_HELM_CHART_VERSION=0.0.39
 
 export FYBRIK_BRANCH="${FYBRIK_BRANCH:-master}"
 export FYBRIK_GITHUB_ORGANIZATION="${FYBRIK_GITHUB_ORGANIZATION:-fybrik}"
 
-if [ $# -gt 1 ]; then
-    echo "Usage: "$0" [open-metadata-version]"
-    exit -1
+if [ $# -gt 3 ]; then
+    echo "Usage: . ./install_OM.sh [install/getFiles/cleanup] [openshift/kind] [open-metadata-version]"
+    return
```

Comment:
when I ran `./install_OM.sh a n s a` I got:

> Usage: . ./install_OM.sh [install/getFiles/cleanup] [openshift/kind] [open-metadata-version]
./install_OM.sh: line 14: return: can only `return' from a function or sourced script
supported OM versions are: 0.12.1
./install_OM.sh: line 49: return: can only `return' from a function or sourced script
about to download installation files to /var/folders/pr/g1rxhbhx6dqbpprfkjtfyfb00000gn/T/tmp.4bfcmnX3
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3508  100  3508    0     0   9853      0 --:--:-- --:--:-- --:--:-- 10022
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1470  100  1470    0     0   4158      0 --:--:-- --:--:-- --:--:--  4212
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   679  100   679    0     0   2286      0 --:--:-- --:--:-- --:--:--  2341

Reply:
The way to run the script is:
```
. ./install_OM.sh param1 param2 ...
```
I will explain it in the documentation